.github/workflows/pull_requests.yml

```diff
@@ -28,21 +28,12 @@ jobs:
       - name: yamllint
         run: make yamllint
 
-  build-sidecar-image:
-    name: Build tailing sidecar image
-    runs-on: ubuntu-20.04
-    steps:
-      - uses: actions/checkout@v2
-      - name: Build tailing-sidecar image
-        working-directory: ./sidecar
-        run: make build
```

Comment:
Can you explain in the PR description why do we remove this? Where is this happening now?